src/test/regress/expected/local_shard_utility_command_execution.out

```diff
@@ -0,0 +1,748 @@
+-- This tests file includes tests for local execution of utility commands.
+-- For now, we this file includes tests only for local execution of
+-- `TRUNCATE/DROP/DDL` commands for all kinds of distributed tables FROM
+-- the coordinator node having regular distributed tables' shards
+-- (shouldHaveShards = on) and having reference table placements in it.
+-- Regression tests for MX will be put into
+-- local_shard_utility_command_execution_mx.sql file.
+RESET log_min_messages;
+RESET client_min_messages;
+SET citus.shard_replication_factor TO 1;
+SET citus.log_remote_commands TO OFF;
+SET citus.log_local_commands TO ON;
+CREATE SCHEMA local_commands_test_schema;
+SET search_path TO local_commands_test_schema;
+SET citus.shard_COUNT TO 32;
```

Comment:
consider using `citus.next_shard_id` for consistent test outputs

Reply:
Do we really need such a consistency as we already normalize test outputs before comparing them ?

Reply:
as far as I know, yes, because we don't normalize all the table names. We normalize hostname/port/taskId/JobID  etc, and some of the table names.

you can see all the rules in `src/test/regress/bin/normalize.sed `.

Reply:
makes sense, applying it